cmd/torpedo/torpedo.go

```diff
@@ -645,6 +672,11 @@ func main() {
 					Usage: "Comma-separated list of tests. [Default:  runs all the tests]",
 					Value: "",
 				},
+				cli.StringFlag{
+					Name:  logLocationCliFlag,
+					Usage: "Path to save logs/artifacts upon failure. Default: /mnt/torpedo-support-dir",
```

Comment:
nit: Usually paths in linux do not have hyphen, you should have underscores - torpedo_support_dir